core/src/main/scala/akka/persistence/cassandra/EventsByTagSettings.scala

```diff
@@ -112,78 +106,86 @@ import com.typesafe.config.Config
 /**
  * INTERNAL API
  */
-@InternalApi private[akka] class CassandraReadJournalConfig(
-    system: ActorSystem,
-    config: Config,
-    writePluginConfig: CassandraJournalConfig)
-    extends NoSerializationVerificationNeeded {
+@InternalApi private[akka] class EventsByTagSettings(system: ActorSystem, config: Config) {
+  import EventsByTagSettings._
+  private val eventsByTagConfig = config.getConfig("events-by-tag")
+  private val log = Logging(system, classOf[EventsByTagSettings])
 
-  val log = Logging(system, classOf[CassandraReadJournalConfig])
+  val eventsByTagEnabled: Boolean = eventsByTagConfig.getBoolean("enabled")
```

Comment:
What about Scala constant style vs regular field, in the Akka `Settings` classes we tend to go with the constant style. Should we align these as well when anyways messing with them?